code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -30,13 +31,24 @@
 
 /obj/item/nullrod/pickup(mob/living/user)
 	. = ..()
-	if(sanctify_force && (!user.mind || !user.mind.isholy))
-		user.adjustBruteLoss(force)
-		user.adjustFireLoss(sanctify_force)
-		user.Weaken(5)
-		user.unEquip(src, 1)
-		to_chat(user, "<span class='warning'>[src] slips out of your grip as you pick it up, bouncing upwards and smacking you in the face!</span>")
-		playsound(get_turf(user), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	if(sanctify_force)
+		if(!user.mind || !user.mind.isholy)
+			user.adjustBruteLoss(force)
+			user.adjustFireLoss(sanctify_force)
+			user.Weaken(5)
+			user.unEquip(src, 1)
+			user.visible_message("<span class='warning'>[src] slips out of the grip of [user] as they try to pick it up, bouncing upwards and smacking [user.p_them()] in the face!</span>", \
+			"<span class='warning'>[src] slips out of your grip as you pick it up, bouncing upwards and smacking you in the face!</span>")
+			playsound(get_turf(user), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+			spawn()
```

Comment:
Shouldn't we be using that new timer system instead?

Reply:
Removed.